tests/PHPStan/Analyser/data/throw-points/ext-ds.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace ThrowPoints\TryCatch;
+
+use PHPStan\TrinaryLogic;
+
+use function PHPStan\Testing\assertVariableCertainty;
+
+function () {
+	$map = new \Ds\Map();
+	try {
+		$map->get('1');
+		$foo = 1;
+	} finally {
+		assertVariableCertainty(TrinaryLogic::createMaybe(), $foo);
+	}
+};
+
+function () {
+	$map = new \Ds\Map();
+	try {
+		$map->get('1', null);
+		$foo = 1;
+	} finally {
+		assertVariableCertainty(TrinaryLogic::createYes(), $foo);
```

Comment:
Any idea why this failed? I've debugged the extension and it returns VoidType in this case so phpstan should know nothing is thrown 🤔